connectivity/mqtt/duplex_utils.c

```diff
@@ -104,10 +105,12 @@ status_t gossip_api_channels_set(mosq_config_t *channel_cfg, char *host, char *r
   char *sub_topic = NULL;
   int sub_topic_len, api_name_len;
   int root_path_len = strlen(root_path);
-  char *api_names[API_NUM] = {"address",          "tag/hashes", "tag/object", "transaction/object",
-                              "transaction/send", "tips/all",   "tips/pair"};
+  char *api_names[] = {"address",  "tag/hashes",         "tag/object",       "transaction/object",
+                       "tryte",    "transaction/single", "transaction/send", "tips/all",
```

Comment:
Do you think is it better to rename `transaction/single` as `transaction`?
I think we could shorten it